robotbase/src/main/java/com/team2357/frc2022/Constants.java

```diff
@@ -12,4 +12,19 @@
  * <p>It is advised to statically import this class (or one of its inner classes) wherever the
  * constants are needed, to reduce verbosity.
  */
-public final class Constants {}
+public final class Constants {
+
+    // CAN IDs
+    public static final int DRIVE_MOTOR_LEFT_1 = 11;
```

Comment:
These should be descriptive to know they're a CAN ID. Perhaps `DRIVE_MOTOR_LEFT_1_CAN_ID` or `CAN_ID_DRIVE_MOTOR_LEFT_1` if you want type-ahead to show you all the CAN IDs easily.

Reply:
As an alternative if you don't want really long names, the constants could be grouped in inner classes.

```
public final class Constants {
  public final class CAN_ID {
    public static final int DRIVE_MOTOR_LEFT_1 = 11;
  }
}
```

Reply:
I was under the impression that we were against using inner classes in the constant file, but I prefer inner classes over the massively long names. I created the inner classes